draft-ietf-ppm-dap.md

```diff
@@ -2623,24 +2621,11 @@ struct {
 } CollectionJobReq;
 ~~~
 
-The named parameters are:
-
 * `query`, the Collector's query. The content of this field depends on the
-  indicated batch mode: for time-interval mode, the `config` field contains the
-  requested time interval ({{time-interval-batch-mode}}); for leader-selected
-  mode, the `config` field is empty, and the Collector gets the aggregate
-  result for the next batch selected by the Leader
-  ({{leader-selected-batch-mode}}).
-
-  Batch modes defined by future documents MUST specify the content of this field;
-  see {{extending-this-doc}} for details.
-
-  The indicated batch mode MUST match the task's batch mode. Otherwise, the
-  Leader MUST abort with error "invalidMessage".
+  indicated batch mode ({{batch-modes}}).
```

Comment:
Please double check that {{batch-modes}} has a reference to {{extending-this-doc}}.

Reply:
It does, in its very first paragraph.